src/createApp.ts

```diff
@@ -940,6 +1019,50 @@ const createApp = compose({
 					};
 				});
 			});
+	},
+
+	start(this: App, { afterCreate, dispatchCurrent, root }: StartOptions = {}): Promise<PausableHandle> {
+		const promise = root ? this.realize(root) : Promise.resolve({ destroy() {} });
+		return promise
+			.then((handle) => {
+				if (!afterCreate) {
+					return handle;
+				}
+
+				return Promise.resolve(afterCreate()).then(() => handle);
+			})
+			.then((realizationHandle) => {
+				// TODO: Ensure there's a test for late binding to the router (e.g. an afterCreate can set a router)
+				const { router } = privateStateMap.get(this);
+
+				if (!router) {
+					return {
+						pause() {},
+						resume() {},
+						destroy() {
+							realizationHandle.destroy();
+						}
+					};
+				}
+
+				const options = {} as RouterStartOptions;
```

Comment:
Nah the problem was that `RouterStartOptions` requires `dispatchCurrent` to be set. To use the default the options simply shouldn't be provided. So this cast was doing the wrong fix.